### PR TITLE
adds support for multiple projects

### PR DESCRIPTION
**Pull request by dwick**
This looks for all the package.json files available and tries to run the cmd on each project.

fixes https://github.com/typicode/husky/issues/7

### CODE REVIEW

**Comment by gajus** [comment]
@typicode Has this feature ever been added?

Specifically, how does this package work with monolithics repositories (i.e. those that contain multiple packages and have multitple husky instances)

**Comment by karlhorky** [comment]
Yeah, it would be cool to have support for monorepos with Lerna or (soon) Yarn workspaces.

File: src/index.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ module.exports = {
   create: function(dir, name, cmd) {
     var filename = dir + '/' + name
     var data =
-        '#!/bin/sh\n'
+        '#!/bin/bash\n'
```
Comment (gillesdemey) [inline src/index.js]: On most *nix systems `/bin/sh` is symlinked to `/bin/bash`

**Comment by SuEric** [comment]
What happened to this?

**Comment by zewa666** [comment]
Any updates on this one?

**Comment by typicode** [comment]
Closing as `1.0` has docs for monorepos and more tooling are available nowadays.

**Pull request closed without merging**